src/NexusMods.App.UI/LeftMenu/Game/GameLeftMenuViewModel.cs

```diff
@@ -87,7 +85,9 @@ private void SelectLoadout(Loadout loadout)
     {
         _logger.LogDebug("Loadout {LoadoutId} selected", loadout.LoadoutId);
         _loadoutGridViewModel.LoadoutId = loadout.LoadoutId;
-        RightContent = _loadoutGridViewModel;
+
+        throw new NotImplementedException("Navigate to workspace");
```

Comment:
`GameLeftMenu*` can probably be removed since the spine doesn't contain individual games but individual loadouts. The original intend behind `GameLeftMenu*` was to select a game on the spine and then switch between loadouts in this menu.